lib/web/scripts/install.go

```diff
@@ -173,12 +176,33 @@ func GetInstallScript(ctx context.Context, opts InstallScriptOptions) (string, e
 //go:embed install/install.sh
 var legacyInstallScript string
 
+var (
+	versionVar = regexp.MustCompile(`(?m)^TELEPORT_VERSION=""$`)
+	suffixVar  = regexp.MustCompile(`(?m)^TELEPORT_SUFFIX=""$`)
+	editionVar = regexp.MustCompile(`(?m)^TELEPORT_EDITION=""$`)
```

Comment:
Nit: I wonder if having some kind of a more specific marker for replacement in the script would be slightly better  from both the potential collision and intent perspective (e.g. `TELEPORT_VERSION="<replace-teleport-version>"` and then do a replacement on the `<replace-teleport-version>`) but no strong opinion either way.

Reply:
The issue is that the script must work out of the box when it's served at `https://cdn.teleport.dev/install.sh`, so I can't put placeholders. Else I would have used gotemplate. I think it's very unlikely that we have another match as we validate both the beginning and end of line + tests will fail if the substitution doesn't happen.